pkg/cmd/prune-backups/prune.go

```diff
@@ -56,54 +56,54 @@ func NewPruneCommand() *cobra.Command {
 	return cmd
 }
 
-func (r *pruneOpts) AddFlags(cmd *cobra.Command) {
+func (r *PruneOpts) AddFlags(cmd *cobra.Command) {
 	flagSet := cmd.Flags()
-	flagSet.StringVar(&r.retentionType, "type", r.retentionType, "Which kind of retention to execute, can either be None, RetentionNumber or RetentionSize.")
+	flagSet.StringVar(&r.RetentionType, "type", r.RetentionType, "Which kind of retention to execute, can either be None, RetentionNumber or RetentionSize.")
 	// the defaults are zero for validation, we inject the real defaults from the periodic backup controller
-	flagSet.IntVar(&r.maxNumberOfBackups, "maxNumberOfBackups", 0, "how many backups to keep when type=RetentionNumber")
-	flagSet.IntVar(&r.maxSizeOfBackupsGb, "maxSizeOfBackupsGb", 0, "how many gigabytes of backups to keep when type=RetentionSize")
+	flagSet.IntVar(&r.MaxNumberOfBackups, "MaxNumberOfBackups", 0, "how many backups to keep when type=RetentionNumber")
```

Comment:
@Elbehery I think your search and replace maybe should not change the argument flags

Reply:
opppsssss .. yeahhhh this is the catch .. thxxxxxxx

Reply:
fixed 👍🏽